antelope_contracts/contracts/stubs/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.16)
+project(stubs_antelope)
+
+find_package(cdt)
+
+set(SOURCES, "")
+
+list(APPEND SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/stub_evm_runtime.cpp
+)
+
+add_contract(  stub_evm_runtime stub_evm_runtime ${SOURCES} )
+target_include_directories( stub_evm_runtime PUBLIC 
+                            ${CMAKE_CURRENT_SOURCE_DIR}/include 
+                            ${CMAKE_BINARY_DIR}/  )
```

Comment:
Which header file(s) is the include directory meant for? There should be a better name used in which the necessary header files are placed to make it clear where they are from.

Better yet, you should use the target if possible to automatically get the public include paths of the target to use here.

Reply:
fixed